MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -208,7 +208,7 @@ public class CarPlayNavigationViewController: UIViewController {
         
         // Update the user puck
         mapView?.updatePreferredFrameRate(for: routeProgress)
-        let camera = MGLMapCamera(lookingAtCenter: location.coordinate, fromDistance: 120, pitch: 60, heading: location.course)
+        let camera = MGLMapCamera(lookingAtCenter: location.coordinate, acrossDistance: 120, pitch: 60, heading: location.course)
```

Comment:
I think the existing values are meant to be altitudes, not viewing distances. See #1815.

Reply:
Dropped this commit